benchmarks-image-service/package.json

```diff
@@ -11,7 +11,7 @@
     "@vx/group": "0.0.189",
     "@vx/pattern": "0.0.189",
     "@vx/scale": "0.0.189",
-    "@vx/stats": "0.0.189",
+    "@vx/stats": "file:stats",
```

Comment:
I think if you are just `require`ing via relative path, I think this can just be omitted entirely.